bin/test

```diff
@@ -26,7 +26,7 @@ for dir in $(find . -type f -name '*_test.go' | grep -v '^./.git/'  | grep -v '^
     GO_TEST_ARGS_RUN="$GO_TEST_ARGS"
```

Comment:
I wonder why we have both `GO_TEST_ARGS` and `GO_TEST_ARGS_RUN`. Not directly related to your changes though.